packages/cms/src/components/icons/icon.tsx

```diff
@@ -69,17 +68,21 @@ export function Icon(props: IconProps) {
         >
           <Box padding={4}>
             <Grid columns={[4, 6]} gap={[1, 1, 2, 3]}>
-              {allIcons.map(([id, src]) => (
-                <Flex
-                  key={id}
-                  direction="column"
-                  align="center"
-                  onClick={(event) => onChange(createPatchFrom(id))}
-                >
-                  <img src={src} width="36" height="36" />
-                  <Radio checked={value === id} readOnly />
-                </Flex>
-              ))}
+              {allIcons.map(([id, src]) => {
```

Comment:
Should be able to use:

```suggestion
              {allIcons.map(([id, GridIcon]) => {
```